hardware/hardware.mk

```diff
@@ -123,13 +131,13 @@ endef
 # bsg_ascii_to_rom, which parses an ASCII file that encodes binary
 # data. Each rom entry is a 32-character string with 1/0 values. Each
 # line is a separate entry. This target generates the verilog for the rom.
-$(HARDWARE_PATH)/%.v: $(HARDWARE_PATH)/%.rom
+$(HARDWARE_MACHINE_PATH)/%.v: $(HARDWARE_MACHINE_PATH)/%.rom
 	python $(BASEJUMP_STL_DIR)/bsg_mem/bsg_ascii_to_rom.py $< \
                bsg_bladerunner_configuration > $@
 
 # This target generates the ASCII file for the ROM. To add entries to
 # the ROM, add more commands below.
-$(HARDWARE_PATH)/bsg_bladerunner_configuration.rom: $(CL_DIR)/Makefile.machine.include
+%/bsg_bladerunner_configuration.rom: %/machine.mk $(HARDWARE_PATH)/Makefile.machine.include 
```

Comment:
However, these changes DO mean that we need to check for changes in machine.mk AND Makefile.machine.include